### PR TITLE
[APB-3855][PT]: Add Create Trust Relationship

### PR DESCRIPTION
**Pull request by JTY009**

### CODE REVIEW

File: app/uk/gov/hmrc/agentclientrelationships/connectors/DesConnector.scala [hunk 1]
```diff
@@ -333,6 +334,11 @@ class DesConnector @Inject()(
           (("idType", JsString("VRN"))) +
           (("relationshipType", JsString("ZA01"))) +
           (("authProfile", JsString("ALL00001")))
+      case Some("TRS") =>
+        request +
+          (("idType", JsString("UTR"))) +
+          (("relationshipType", JsString("ZA01"))) +
+          (("authProfile", JsString("ALL00001")))
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentclientrelationships/connectors/DesConnector.scala]: are these same as above for VRN case ?

File: it/uk/gov/hmrc/agentrelationships/connectors/DesConnectorSpec.scala [hunk 2]
```diff
@@ -261,6 +283,12 @@ class DesConnectorSpec
       await(desConnector.deleteAgentRelationship(Vrn("foo"), Arn("bar"))).processingDate should not be null
     }
 
+    "delete relationship between agent and client and return 200 for Utr service" in {
+      givenAgentCanBeDeallocatedInDes(Vrn("foo"), Arn("bar"))
```
Comment (sureshhmrc) [inline it/uk/gov/hmrc/agentrelationships/connectors/DesConnectorSpec.scala]: shouldn't we use Utr here instead of Vrn?

Comment (JTY009) [reply it/uk/gov/hmrc/agentrelationships/connectors/DesConnectorSpec.scala]: Yep

File: app/uk/gov/hmrc/agentclientrelationships/connectors/DesConnector.scala (hunk 1 above)
Comment (JTY009) [reply app/uk/gov/hmrc/agentclientrelationships/connectors/DesConnector.scala]: In theory

**Pull request merged**